src/main/resources/rules/learningRecord.drl

```diff
@@ -0,0 +1,97 @@
+import com.deloitte.elrr.elrrconsolidate.dto.UserCourse;
```

Comment:
this rule seems like a subset of what is already happening in message listener. We need to make the indent code tight in listener, and then do all the activity processing in here.